MSOffice2011Updates/MSOffice2011UpdateInfoProvider.py

```diff
@@ -166,6 +166,13 @@ def get_version(self, item):
     def value_to_os_version_string(self, value):
         """Converts a value to an OS X version number"""
         #pylint: disable=no-self-use
+
+        # Map string type for both Python 2 and Python 3.
+        try:
+            _ = basestring
```

Comment:
I'm not a fan of this at all, and we should come up with a better alternative to address this in the future. In a Py3 only world, there won't be any basestrings anymore, so I can live with it until we rip them all out. But this construction bothers me immensely.

Reply:
If you are going to consider removing TextWrangler because it's dead then give the same consideration to Office 2011.

Reply:
@nmcspadden What's your primary objection to this construction?